pvr.mediaportal.tvserver/resources/language/resource.language.de_de/strings.po

```diff
@@ -64,6 +64,10 @@ msgctxt "#30011"
 msgid "Enable old series recording dialog"
 msgstr "Aktiviere Dialog für alte Serien-Aufnahmen"
 
+msgctxt "#30012"
+msgid "Default recording lifetime"
+msgstr ""
```

Comment:
Please drop the changes to the German string resource from this PR. All languages except en_gb will be handled through Transifex (and your changes would get lost, then).

Reply:
And also drop all others (en_us, ...). Same problem as with de_de.